src/App.css

```diff
@@ -17,6 +17,9 @@
 .App-intro {
   font-size: large;
 }
+#widget-container {
```

Comment:
maybe this should be called github-widget-container. so it won't clash with other potential ids. Best practice to just avoid ids anyway. go with classes.